Gemfile

```diff
@@ -23,3 +23,6 @@ gem "rubyzip", "~> 1.1.7", :group => :build
 gem "gems", "~> 0.8.3", :group => :build
 gem "rack-test", :require => "rack/test", :group => :development
 gem "flores", "~> 0.0.6", :group => :development
+gem "pleaserun"
+gem "mustache"
```

Comment:
I don't see Mustache invoked in this PR. Pleaserun uses it, but lists it as a dependency. Do we need it listed here?

Same question for Clamp.